dubbo-common/src/main/java/org/apache/dubbo/common/utils/NetUtils.java

```diff
@@ -191,7 +192,7 @@ static InetAddress normalizeV6Address(Inet6Address address) {
 
     public static String getLocalHost() {
         InetAddress address = getLocalAddress();
-        return address == null ? LOCALHOST : address.getHostAddress();
+        return address == null ? Constants.LOCALHOST_VALUE : address.getHostAddress();

```

Comment:
I think Constants.LOCALHOST should be good value looks like extra here. What do you say?

Reply:
thanks,I am not sure about this, I feel no difference.

Reply:
LOCALHOST is good enough, LOCALHOST_VALUE sound extra here. Having said this, it is very minor both should be ok. I leave it to you. If any has any suggestion it can be modified ltr. :)